lib/routes/nm/activateConnection.ts

```diff
@@ -0,0 +1,36 @@
+import * as _ from 'lodash';
+import * as uuid from 'uuid';
+
+export default {
+	'connect-network': (serviceInstance: any) => (req: any, res: any) => {
+		const connection = req.body.value;
+		console.log('Connecting to netwotk', _.omit(connection, 'passphrase'));
+		// /org/freedesktop/NetworkManager/Settings/16 /org/freedesktop/NetworkManager/ActiveConnection/13
+		// serviceInstance.ActivateConnection('/org/freedesktop/NetworkManager/Settings/16', '/org/freedesktop/NetworkManager/Devices/2', '/', (err, activeConnection) => {
+		serviceInstance.ActivateConnection(`/org/freedesktop/NetworkManager/Settings/${req.body.pathN}`, '/', '/', (err, activeConnection) => {
+			if (err) {
+				console.error('err', err);
+				return res
+					.status(500)
+					.json(err);
+			}
+			console.log('YASS!', activeConnection);
+			return res
+				.status(200)
+				.send({
+					headers: {
+						'Content-Type': 'text/html',
+					},
+					body: `Connected to network: ${JSON.stringify(activeConnection)}`,
+				});
+		});
+	},
+};
+
+function stringToArrayOfBytes(str) {
```

Comment:
This is not used and not exported, it is also defined in  lib/routes/nm/addAndActivateConnection.ts where it is used.